requirements.txt

```diff
@@ -4,19 +4,19 @@
 #
```

Comment:
Still no iteration_utilities in this file. Double-check how to add a requirement: https://github.com/DeborahAlmeida/pokebattle#adding-new-pypi-libs

Reply:
I'm not using iteration_utilities anymore. But I'm going to check it, thank you!

Reply:
Thank you, I realized where I was doing wrong.